src/components/button/_button-base.scss

```diff
@@ -111,7 +120,8 @@ $md-mini-fab-padding: 8px !default;
     padding: $padding 0;
   }
 
-  &.md-button-focus {
-    background-color: md-color($md-accent, 600);
+  // The focus shading for FABs needs to match the FAB's circular shape.
+  &.md-button-focus:after {
+    border-radius: 50%;
```

Comment:
Would it be better to use `border-radius: inherit` in the declaration above instead? This way it will also respect the border radii for normal flat + raised buttons.